packages/nextjs/pages/index.tsx

```diff
@@ -209,37 +217,125 @@ const Home: NextPage<{ stats: Stats }> = ({ stats }) => {
 
       {/* Supporting Devs*/}
       <div className="bg-base-300">
-        <div className="mx-auto lg:max-w-[1980px] bg-none lg:bg-[url('/assets/funding.png')] md:[background-position-x:40vw] md:bg-contain bg-no-repeat xl:bg-right">
+        <div className="mx-auto lg:max-w-[1980px] bg-none lg:bg-[url('/assets/support-high-impact-devs.png')] md:[background-position-x:45vw] [background-position-y:50%] md:bg-auto bg-no-repeat ">
           <div className="container max-w-[90%] lg:max-w-6xl m-auto py-16 lg:py-20 lg:px-12 flex flex-col lg:flex-row items-center gap-5 lg:gap-0">
             <div className="text-center lg:text-left sm:w-1/2">
-              <h2 className="text-2xl lg:text-5xl font-semibold my-0 mb-6">
+              <h2 className="text-2xl lg:text-5xl font-semibold my-0 mb-2 lg:mb-6">
                 Supporting
                 <br /> up-and-coming <br />
                 high-impact devs
               </h2>
+              {/* Mobile: Set background image as regular image in a div */}
+              <div className="lg:hidden mb-2">
+                <Image
+                  src="/assets/support-high-impact-devs.png"
+                  alt="Supported list of devs"
+                  width={1115}
+                  height={800}
+                />
+              </div>
+              <p className="lg:w-4/5 m-0 mb-3">
+                Open Developer Streams are a unique way to fund development and give developers at the edges the freedom
+                to build what they think is most important.
+              </p>
               <p className="lg:w-4/5 m-0 mb-3">
-                BuidlGuidl streams to developers using our new cohort streams and other custom smart contracts.
+                Their smart contracts get replenished monthly and allows them to withdraw funds whenever they like by
+                submitting a few sentences about the work or a PR.
               </p>
               <p className="lg:w-4/5 m-0 mb-6">
-                Our goal is to enrich open learning within the Ethereum developer ecosystem.
+                This approach produces novel open source solutions and a vibrant learning environment.
               </p>
+              <p className="lg:mb-3 mt-12 lg:mt-8 text-sm">SUPPORTED BY</p>
+              <div className="flex flex-col gap-6 items-center lg:items-start">
+                <Image src="/assets/ef-logo.png" alt="EF logo" width={200} height={200} />
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      {/* Cohorts*/}
+      <div className="bg-base-100">
+        <div className="mx-auto lg:max-w-[1980px]">
```

Comment:
maybe we could use `lg:max-w-6xl` instead of custom value because that's what we using in other sections like `scaffold-eth 2` , `Test your knowledge` : 

https://github.com/BuidlGuidl/buidlguidl.com/blob/8040a323ba976d535ffef8c2e6e5f7895abd74f8/packages/nextjs/pages/index.tsx#L64

Reply:
Updated! 🙌